explorer/explorertypes.go

```diff
@@ -67,9 +67,16 @@ type TxInfo struct {
 }
 
 type TicketInfo struct {
-	TicketMaturity int64
-	PoolStatus     string
-	SpendStatus    string
+	TicketMaturity               int64
+	TimeTillMaturity             float64 //TimeTillMaturity represents the time before a particular ticket reaches maturity
```

Comment:
For consistency with the rest of the codebase and , please use a space after the slashes in the comment